perk-store-webapps/src/main/webapp/css/main.less

```diff
@@ -888,7 +889,12 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
   .buyButton{
     margin-top: 10px;
   }
-
+  .disabledBuyButton{
+    background-color: #E0E0E0 !important;
+  }
+  .VuetifyApp .v-btn--disabled {
+    pointer-events: auto;
+  }
```

Comment:
please change this, the CSS you add will apply to all the v-btn in the platform, you have to add these CSS in the parent class

Reply:
it's just to make its title appears!

Reply:
done!